src/client/app/types/react-plotlyjs-ts/index.d.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference types="react" />
```

Comment:
<https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html> gives limited circumstances where reference should be used. Wanted to check that this is the best way to do this.